src/challenges.js

```diff
@@ -28,16 +44,32 @@ const cacheFunction = cb => {
 
 /* ======================== Recursion Practice ============================ */
 const reverseStr = str => {
```

Comment:
You have the right idea but we are expecting you to use recursion for this function.